### PR TITLE
PLAT-86414: Fix syntax highlighting

### PR DESCRIPTION
**Pull request by RenukaAtale**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [ ] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

File: docs/migration/enyo/migrate-fittables.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Fittable example, the `FittableColumnsLayout` component behaves as a row (the ch
 
 This is a simple example of creating a `FittableColumnsLayout` style with Enact and flexbox. This
 example uses inline `style` tags, though using CSS classes would be preferable.
-```
+```html
```
Comment (webOS101) [inline docs/migration/enyo/migrate-fittables.md]: This is js, not html

File: docs/migration/enyo/migrating-enyo-apps.md [hunk 2]
```diff
@@ -42,14 +42,14 @@ In general, the `content` property is now handled by the implicit `children` pro
...
-    { name: 'MyButton', kind: Button, small: true, content: 'Click Me!'}
+```json
+{ name: 'MyButton', kind: Button, small: true, content: 'Click Me!'}
 ```
 
 In Enact, the same effect is achieved like this:
 
-```
-    <Button size="small">Click Me!</Button>
+```html
```
Comment (webOS101) [inline docs/migration/enyo/migrating-enyo-apps.md]: Same here.

**Comment by ybsung** [comment]
Hello @webOS101 
Would it be better to fix [virtual-list-scroller.md](https://github.com/enactjs/enact/blob/develop/packages/moonstone/docs/virtual-list-scroller.md) here or create another issue?

**Comment by webOS101** [comment]
@ybsung @RenukaAtale It would be good to fix as part of this, definitely!

**Pull request merged**